Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -9495,10 +9495,13 @@ def get_current_time(time_zone=0):
 
 def filter_incidents_by_duplicates_and_limit(incidents_res, last_run, fetch_limit, id_field):
     """
-    Remove duplicate incidents from response and returns the incidents till limit
+    Removes duplicate incidents from response and returns the incidents till limit.
+    The function should be called after getting the get-incidents API response,
```

Comment:
```suggestion
    The function should be called in case using look-back greater than zero after getting the get-incidents API response,
```

am i right here? I am unsure

Reply:
It should not disturb the regular fetch process and can be used also when `look_back=0` for filtering by limit.